tests/imposter/private-api-config.yaml

```
@@ -22,7 +22,7 @@ resources:
   - method: GET
     path: /question
     requestHeaders:
-      session-id: insufficient-questions
+      session-id: "insufficient-questions"
```

```suggestion
      session-id: insufficient-questions
```
Looks like we don't use quotes for string in this file so this is stylisically inconsistent.